plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
         "Extension": "extension",
         "Filesystem": "Filesystem",
         "GetInvolved": "If you like what you see, you can %1$sget involved%2$s.",
-        "GoBackAndDefinePrefix": "Go back and define a Prefix for the Matomo Tables",
+        "GoBackAndDefinePrefix": "Go back and define a prefix for the Matomo tables",
         "HappyAnalysing": "Happy analysing!",
         "Installation": "Installation",
         "InformationalResults": "Informational results",
-        "CopyBelowInfoForSupport": "Copy or download the below info in case our support team asks you for this information.",
+        "CopyBelowInfoForSupport": "Copy or download the info below, in case our support team asks you for this info.",
         "CopySystemCheck": "Copy system check",
         "DownloadSystemCheck": "Download system check",
         "InstallationStatus": "Installation status",
-        "InsufficientPrivilegesHelp": "You can add these privileges by using a tool such as phpMyAdmin or by executing the right SQL queries. If you don't know how to do these things, please ask your sysadmin to grant these privileges for you.",
-        "InsufficientPrivilegesMain": "Either the Database does not exist (and could not be created), or the specified User has insufficient privileges. Database user must have the following privileges: %s",
+        "InsufficientPrivilegesHelp": "These privileges can be added by phpMyAdmin, or by executing the right SQL queries. If you don't know how, please ask your sysadmin to do so.",
+        "InsufficientPrivilegesMain": "Either the database does not exist (and could not be created), or the specified user has insufficient privileges. A database user must have the following privileges: %s",
         "InvalidStateError": "Error: Matomo is already installed. %1$s Go back %2$s to Matomo%3$s.",
-        "JsTagArchivingHelp1": "For medium and high traffic websites there are certain optimizations that should be made to help Matomo run faster (such as %1$ssetting up auto-archiving%2$s).",
+        "JsTagArchivingHelp1": "For medium and high traffic website certain optimizations help Matomo run faster (such as %1$ssetting up auto-archiving%2$s).",
         "JSTracking_EndNote": "Note: After the installation process, you can generate customized tracking code in the %1$sTracking Code%2$s admin section.",
-        "JSTracking_Intro": "To track your web traffic with Matomo you need to make sure some extra code is added to each of your webpages.",
+        "JSTracking_Intro": "To track your web traffic with Matomo you need to make sure a tracking code is added to each of your webpages.",
```

Comment:
"a tracking code" doesn't quite make sense. Suggest reverting this one as specifying tracking code isn't much clearr than the existing.

Reply:
Is "some extra code" a quantity, or is it (for some reason) an unspecified piece of code.
If this isn't the JavaScript code I don't know what is going on, which is still better than being mislead by erroneous translations.